### PR TITLE
[New Rule] Complementary Rules for Recent REvil TTPs

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/1328

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/windows/defense_evasion_disabling_windows_defender_powershell.toml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+[metadata]
+creation_date = "2021/07/07"
+maturity = "production"
+updated_date = "2021/07/07"
+
+[rule]
+author = ["Elastic"]
+description = "Identifies use of Set-MpPreference powershell command to disable or weaken certain Windows Defender preferences."
```
Comment (devonakerr) [inline rules/windows/defense_evasion_disabling_windows_defender_powershell.toml:8]: Instead of preferences, could be say "security settings"?

**Review by devonakerr — APPROVED** [review]
One comment but consider it informational for purposes of approval.

File: rules/windows/defense_evasion_disabling_windows_defender_powershell.toml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+false_positives = ["Planned Windows Defender configuration preferences changes."]
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Enable Host Network Discovery via Netsh"
```
Comment (devonakerr) [inline rules/windows/defense_evasion_disabling_windows_defender_powershell.toml:14]: I think this may be a copy/paste issue

**Review by devonakerr — COMMENTED** [review]
Couple small things to tweak.

File: rules/windows/defense_evasion_disabling_windows_defender_powershell.toml (hunk 1 above)
Comment (Samirbous) [reply rules/windows/defense_evasion_disabling_windows_defender_powershell.toml]: changed.

File: rules/windows/defense_evasion_disabling_windows_defender_powershell.toml (hunk 2 above)
Comment (Samirbous) [reply rules/windows/defense_evasion_disabling_windows_defender_powershell.toml]: 👍 corrected

**Review by devonakerr — APPROVED** [review]
LGTM

File: rules/windows/defense_evasion_disabling_windows_defender_powershell.toml [hunk 3]
```diff
@@ -0,0 +1,49 @@
+[metadata]
+creation_date = "2021/07/07"
+maturity = "production"
+updated_date = "2021/07/07"
+
+[rule]
+author = ["Elastic"]
+description = "Identifies use of Set-MpPreference powershell command to disable or weaken certain Windows Defender settings."
```
Comment (peasead) [inline rules/windows/defense_evasion_disabling_windows_defender_powershell.toml:8]:
```suggestion
description = "Identifies use of the Set-MpPreference PowerShell command to disable or weaken certain Windows Defender settings."
```

File: rules/windows/defense_evasion_enable_network_discovery_with_netsh.toml [hunk 4]
```diff
@@ -0,0 +1,49 @@
+[metadata]
+creation_date = "2021/07/07"
+maturity = "production"
+updated_date = "2021/07/07"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies use of the netsh.exe to enable host discovery via the network. Attackers will use this command line tool to
+weaken the host firewall settings.
```
Comment (peasead) [inline rules/windows/defense_evasion_enable_network_discovery_with_netsh.toml:10]:
```suggestion
Identifies use of the netsh.exe program to enable host discovery via the network. Attackers can use this command-line tool to
weaken the host firewall settings.
```

File: rules/windows/defense_evasion_execution_windefend_unusual_path.toml [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+creation_date = "2021/07/07"
+maturity = "production"
+updated_date = "2021/07/07"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies an instance of a Windows trusted program that is known to be vulnerable to DLL Search Order Hijacking
+starting after being renamed or from a non-standard path. This is uncommon behavior and may indicate an attempt to evade
+defenses via side loading a malicious DLL within the memory space of one of those processes.
```
Comment (peasead) [inline rules/windows/defense_evasion_execution_windefend_unusual_path.toml:11]:
```suggestion
Identifies a Windows trusted program that is known to be vulnerable to DLL Search Order Hijacking
starting after being renamed or from a non-standard path. This is uncommon behavior and may indicate an attempt to evade
defenses via sideloading a malicious DLL within the memory space of one of those processes.
```

File: rules/windows/defense_evasion_execution_windefend_unusual_path.toml [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+"""
+false_positives = ["Microsoft Antimalware Service Executable installed on non default installation path."]
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential DLL SideLoading via Microsoft Antimalware Service Executable"
```
Comment (peasead) [inline rules/windows/defense_evasion_execution_windefend_unusual_path.toml:18]:
```suggestion
name = "Potential DLL Side-Loading via Microsoft Antimalware Service Executable"
```

**Review by peasead — APPROVED** [review]
Few comments, but LGTM.

**Comment by eklike100** [comment]
Sorry I am off topic but I really want to reach out to @Samirbous to check if he can contribute any rule for PPID spoofing (T1134.004). Fireeye has one python script which uses pywintrace to log events from ETW which is available [here](https://github.com/countercept/ppid-spoofing/blob/master/detect-ppid-spoof.py)

**Comment by Samirbous** [comment]
@eklike100 thanks, we will push a rule soon.

**Pull request merged**